zest/releaser/pypi.py

```diff
@@ -316,11 +315,13 @@ def no_input(self):
         mixed case and specify 0, false, no or off for boolean False,
         and 1, on, true or yes for boolean True.
         """
-        default = False
-        if self.config is None:
-            return default
-        try:
-            result = self.config.getboolean('zest.releaser', 'no-input')
-        except (NoSectionError, NoOptionError, ValueError):
-            return default
+        return self.__get_boolean('zest.releaser', 'no-input')
+
+    def __get_boolean(self, section, key):
```

Comment:
The `__get_boolean` method should accept a default value. Currently you make the default False. In `want_release` we want True as default, which is not happening now.

Also, I would rename this to `_get_boolean` or `_getboolean`. A double underscore indicates a special method, which is not the case here.